pandas/tests/frame/methods/test_cov_corr.py

```diff
@@ -175,6 +175,15 @@ def test_corr_int(self):
         df3.cov()
         df3.corr()
 
+    @td.skip_if_no_scipy
+    @pytest.mark.parametrize("meth", ["pearson", "spearman", "kendall"])
+    def test_corr_axes(self, meth):
+        # https://github.com/pandas-dev/pandas/issues/35002
+        df = pd.DataFrame(np.random.normal(size=(10, 4)))
```

Comment:
put axis labels that are differnt for rows / columns and this should fail (need to handle that)